ocs_ci/utility/utils.py

```diff
@@ -411,13 +411,20 @@ def run_cmd(cmd, secrets=None, timeout=600, ignore_error=False, **kwargs):
         timeout=timeout,
         **kwargs
     )
-    log.debug(f"Command output: {r.stdout.decode()}")
-    if r.stderr and not r.returncode:
-        log.warning(f"Command warning: {mask_secrets(r.stderr.decode(), secrets)}")
+    if len(r.stdout) > 0:
+        log.debug(f"Command stdout: {r.stdout.decode()}")
+    else:
+        log.debug("Command stdout is empty")
+    masked_stderr = mask_secrets(r.stderr.decode(), secrets)
+    if len(r.stderr) > 0:
+        log.warning(f"Command stderr: {masked_stderr}")
```

Comment:
log.debug() instead?

Reply:
This is done on purpose for better visibility, moreover the previous code also reported this via WARNING log level.